src/lambdapi.ml

```diff
@@ -106,6 +106,7 @@ let _ =
   (* Parse command line arguments while accumulating all files. *)
   let usage = Printf.sprintf "Usage: %s [OPTIONS] [FILES]" Sys.argv.(0) in
   let files = ref [] in
+  print_string "This a work in progress for adding implicit arguments to lambdapi \n";
```

Comment:
Should be removed.

Reply:
Of course, it's going to be removed for the real PR. It's just currently useful for me when I compare the behaviours my changes on lambdapi and the official lambdapi, in order to see immediately which is which. Especially if I forget a make install...

Reply:
I think you lose all the expected properties of this line if you do so. Since this line will be on every branch, so would not indicate you if the installed version is the experimental one or not anymore.